_pages/cfp.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
the details are out of date there though now, and since we mention it I'd rather have up to date linking